health-services/project-factory/src/server/utils/projectCampaignUtils.ts

```
@@ -0,0 +1,565 @@
+    return campaignProjects.reduce((acc: any, curr: any) => {
+        acc[curr.boundaryCode] = curr;
+        return acc;
+    }, {});
+}
+
+async function fetchCampaignProjectsData(
+    campaignNumber: string,
+    searchActiveOnly: boolean,
+    boundaryCodes: string[] = [],
+    parentBoundaryCodes: string[] = [],
+    fetchCountOnly: boolean = false
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)